.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Merge request
+about: Default template for a pull request
+---
+
+*Please indicate here a summary of pull request content or if related issue exist, please indicate issue reference here*
+
+## Checkpoints
+
+The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
+If one or more of these checkpoints are out of scope, please write below why they are.
+
+- [ ] Code is intelligible enought
+- [ ] If there is FIXME in code, these is an associated issue and issue reference is indicate in FIXME
+- [ ] If there is TODO or NOTE, or HACK in code, date and developper initials are present
```

Comment:
`If there is TODO or NOTE, or HACK in code, date and developper initials are present` => `If there are TODO, NOTE or HACK in code, date and developer initials are present`